python/cuml/cuml/svm/linear_svr.py

```diff
@@ -138,6 +139,22 @@ def __init__(self, **kwargs):
 
         super().__init__(**kwargs)
 
+    @classmethod
+    def _params_from_cpu(cls, model):
+        return {
+            "loss": model.loss,
+            "epsilon": model.epsilon,
+            **super()._params_from_cpu(model),
+        }
+
+    def _params_to_cpu(self):
+        return {
+            "loss": self.loss,
+            "epsilon": self.epsilon,
+            "dual": False,  # cuML LinearSVR is primal
```

Comment:
No reason to set this - primal/dual just relates to how the solver handles the optimization, not to the values of the computed coefficients. Leaving this as the default ("auto") makes more sense.

```suggestion
```

Reply:
Addressed in d9b818eb1ac392b3f4c115a2825f60d9f75e246c .